src/Microsoft.DocAsCode.Build.ManagedReference/FillReferenceInformation.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.DocAsCode.Build.ManagedReference
     [Export(nameof(ManagedReferenceDocumentProcessor), typeof(IDocumentBuildStep))]
     public class FillReferenceInformation : BaseDocumentBuildStep, ICanTraceContextInfoBuildStep
     {
+        private IEnumerable<SourceInfo> _lastContextInfo;
         private Dictionary<string, SourceInfo> _items = new Dictionary<string, SourceInfo>();
```

Comment:
storing the whole set of itemviewmodel is already in the stable bits.
if only store what the step needs, then it is all the itemviewmodel that the articles reference, but considering the following case: A references B, which would lead to a dependnecyitem: A->B. now if A updates to reference B and C, now we don't have C's model in our cache because C isn't changed.